src/main/java/org/folio/rest/impl/UserTenantsAPI.java

```diff
@@ -36,24 +39,42 @@ public void getUserTenants(String userId, String username, String tenantId, int
     String okapiTenantId = TenantTool.tenantId(okapiHeaders);
     Criterion criterion = new Criterion().setLimit(new Limit(limit)).setOffset(new Offset(offset));
     addWhereClauseArgumentsToCriterion(userId, username, tenantId, criterion);
+    logger.debug("Trying to get user-tenant records with criterion: {}.", criterion);
 
     userTenantService.fetchUserTenants(okapiTenantId, criterion, vertxContext.owner())
-      .onSuccess(res -> asyncResultHandler.handle(
-        succeededFuture(GetUserTenantsResponse.respond200WithApplicationJson(res))))
-      .onFailure(cause -> asyncResultHandler.handle(
-        succeededFuture(GetUserTenantsResponse.respond500WithTextPlain(cause.getMessage()))));
+      .onSuccess(res -> {
+        logger.info("Number of existing user-tenant records: {}.", res.getTotalRecords());
```

Comment:
Lets use debug here for get request, because it will be called by mod-authtoken often(mod-authtoken will have a chache for it but nevertheless).

Reply:
Changed logging level to debug.